org.eclipse.dawnsci.remotedataset.server/src/org/eclipse/dawnsci/remotedataset/server/slice/SliceRequest.java

```diff
@@ -170,9 +172,17 @@ private ILazyDataset getLazyDataset(String path, String dataset) throws Exceptio
 		
 		final File   file = new File(path); // Can we see the file using the local file system?
 		if (!file.exists()) throw new IOException("Path '"+path+"' does not exist!");
-		ServiceHolder.getLoaderService().clearSoftReferenceCache();
+//		ServiceHolder.getLoaderService().clearSoftReferenceCache();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)